ui/src/app/layout/topics/schema-section.component.ts

```diff
@@ -0,0 +1,112 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { SchemaMetadata, Topic, TopicsService, TopicSubscription } from '../../shared/services/topics.service';
+import { take } from 'rxjs/operators';
+import { EnvironmentsService, KafkaEnvironment } from '../../shared/services/environments.service';
+import { ToastService } from '../../shared/modules/toast/toast.service';
+import { combineLatest, Observable } from 'rxjs';
+import { TranslateService } from '@ngx-translate/core';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+    selector: 'app-schema-section',
+    templateUrl: './schema-section.component.html',
+    styleUrls: ['./schema-section.component.scss']
+})
+export class SchemaSectionComponent implements OnInit {
+
+    @Input() topic: Observable<Topic>;
+
+    @Input() topicSubscribers: Observable<TopicSubscription[]>;
+
+    @Input() selectedEnvironment: Observable<KafkaEnvironment>;
```

Comment:
Should not be an input to this component, as this can be queried from the `environmentService` (which is injected via constructor already).